planner/core/cbo_test.go

```diff
@@ -678,9 +678,9 @@ func (s *testAnalyzeSuite) TestCorrelatedEstimation(c *C) {
 			"  └─MaxOneRow_13 1.00 root ",
 			"    └─Projection_14 0.80 root concat(cast(t1.a), \",\", cast(t1.b))",
 			"      └─IndexLookUp_21 0.80 root ",
-			"        ├─IndexScan_18 1.00 cop table:t1, index:c, range: decided by [eq(t1.c, test.t.c)], keep order:false",
+			"        ├─IndexScan_18 1.25 cop table:t1, index:c, range: decided by [eq(t1.c, test.t.c)], keep order:false",
 			"        └─Selection_20 0.80 cop eq(t1.a, test.t.a)",
-			"          └─TableScan_19 1.00 cop table:t, keep order:false",
+			"          └─TableScan_19 1.25 cop table:t, keep order:false",
```

Comment:
For reviewers: this row count change is caused by the fact that index on column `c` exists, so we would not analyze column `c`, hence estimated NDV of column `c` would be `ndv = float64(ds.statisticTable.Count) * distinctFactor`, which is 8, not 10. This behavior is different with master branch, because after https://github.com/pingcap/tidb/pull/9315, we would analyze column `c` even if index on column `c` exists.